js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -100,7 +103,7 @@ class RequestsPage extends Component {
           <div className={ styles.noRequestsMsg }>
             <FormattedMessage
               id='signer.requestsPage.noPending'
-              defaultMessage='There are no requests requiring your confirmation.'
+              defaultMessage='There are no transactions requiring your confirmation.'
```

Comment:
Your password in signer might be required for couple of things (as Rob mentioned):
1. Sign transaction `eth_sendTransaction`
2. Sign transaction for offline use `eth_signTransaction`
3. Sign aribtrary data `eth_sign`
4. Decrypt arbitrary data `parity_decryptMessages`